src/main/java/nextstep/favorite/application/exception/FavoriteException.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.favorite.application.exception;
+
+import nextstep.exception.ApplicationException;
+import org.springframework.http.HttpStatus;
+
+public class FavoriteException extends ApplicationException {
+
+    private final static String NOT_FOUND = "즐겨찾기를 찾을 수 없습니다.";
+    private final static String NOT_CREATED = "즐겨찾기 추가에 실패했습니다.";
```

Comment:
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)에 따라 접근 제어자의 순서를 변경하면 좋을 것 같아요
```suggestion
    private static final String NOT_FOUND = "즐겨찾기를 찾을 수 없습니다.";
    private static final String NOT_CREATED = "즐겨찾기 추가에 실패했습니다.";
```